spec/io/io_spec.rb

```diff
@@ -86,6 +86,20 @@
       end
     end
 
+    context "#from_html" do
+      let(:path) { path = "https://en.wikipedia.org/wiki/Table_(information)" }
```

Comment:
So... Each time somebody runs the specs, the data would be fetched from internet? It could not be done this way for several reasons:
* slow;
* what if my internet is down currently? or wikipedia is down?
* what if the wikipedia page will be changed? the test will fail for no internal reason

We use webmock/vcr for such cases, but for this particular test it is probably wiser to just save some test HTMLs into spec/fixtures, and use them.